case-api-client/src/main/java/uk/gov/ons/ctp/integration/caseapiclient/caseservice/model/RmCaseDTO.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.ons.ctp.integration.caseapiclient.caseservice.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import lombok.Data;
+
+@Data
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class RmCaseDTO {
+
+  private String caseRef;
+
+  private Date createdDateTime;
+
+  private Date lastUpdated;
+
+  private List<EventDTO> caseEvents;
```

Comment:
should we just have one `CaseEventDTO` ? not sure about this or the `EventDTO` better check with @philwhiles

Reply:
RmCaseDTO and CaseEventDTO are the types we use to get the case from RM.
CC would not return raw RM DTOs to its client (Serco in Census, and now our own UI), as it may wish to add/remove fields etc.
The CC CaseEventDTO will be replaced by one of the following tickets in this epic with CaseInteractionDTO anyway.